301-azure-relay-create-authrule-namespace-and-wcfrelay/azuredeploy.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "namespaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Azure Relay namespace"
+      }
+    },
+    "namespaceAuthorizationRuleName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Namespace AuthorizationRule"
+      }
+    },
+    "wcfrelayName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the WcfRelay"
+      }
+    },
+	"wcfRelayType": {
+      "type": "string",
+      "allowedValues": [
+        "NetTcp",
+        "Http"
+      ],
+      "metadata": {
+        "description": "WCF Relay Type. It could be any of the types: NetTcp/Http"
+      }
+    },
+    "wcfrelayAuthorizationRuleName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the WcfRelay AuthorizationRule"
+      }
+    }
+  },
+  "variables": {
+    "location": "[resourceGroup().location]",
+    "apiVersion": "2017-04-01",
```

Comment:
remove this and use a literal value

Reply:
Actually we have used the similar way in the Service Bus and EventHub.

Reply:
It's not optional, it prevents schema validation from being done.

Reply:
resolved.